README.md

```diff
@@ -1,2 +1,58 @@
 # publicly-available-microscopy-data
-Compiling and describing publicly available microscopy datasets
+## Purpose
```

Comment:
```suggestion

## Purpose
```

New lines between headings

Reply:
Added in 1a10c619bd1a83be992e039f347a46aae6021a67.